tests/Application/.env.test

```diff
@@ -1,3 +1,23 @@
-APP_SECRET='ch4mb3r0f5ecr3ts'
+# This file is a "template" of which env vars needs to be defined in your configuration or in an .env file
```

Comment:
Shouldn't this file be like this: https://github.com/Sylius/PluginSkeleton/blob/master/tests/Application/.env.test ?

Reply:
`APP_SECRET` could be random.
`KERNEL_CLASS` is used by PHPunit and was defined at `phpunit.xml.dist`.

I believe https://github.com/Sylius/PluginSkeleton/blob/master/tests/Application/.env.test not final and was committed accidentally or something like this.

Reply:
Hmm, okay